enhancements/update/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,205 @@
+---
+title: cincinnati-graph-data-package-distribute-and-consume-in-a-container-image-repository
+authors:
+  - "@steveeJ"
+reviewers:
+  - @openshift/openshift-team-cincinnati-maintainers
+  - @openshift/openshift-team-cincinnati
+  - @openshift/team-cincinnati-operator
+approvers:
+  - @crawford
+  - @LalatenduMohanty
+  - @sdodson
+creation-date: 2020-05-07
+last-updated: 2020-05-07
+status: provision
+see-also:
+-
+replaces: []
+superseded-by: []
+---
+
+# cincinnati-graph-data: package, distribute, and consume in a container image repository
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+### Content trust
+Trust in the container image registry and in the container image content is possibly an issue if we follow an approach where Cincinnati uses a mutable tag, e.g. `latest` tag to receive the most recent content from the container registry.
+Assuming we're using Quay as an image registry, [which does not support Docker Content Trust](https://support.coreos.com/hc/en-us/articles/115000289114-Docker-Content-Trust-support).
+If we decide to ensure image integrity, which means do we have that do not further complicate the workflow?
+* An option is to package the signatures as a container image as well, and publish them in a separate repository, or in the same repository under a well-known tag e.g. `latest-signature`.
+  This option is currently described in this enhancement.
+
+### Product name for the CPaaS integration
+Which product name should this image be released under?
+Further note: the repository for the graph-data images will not be semantically versioned.
+
+### Repository location and name
+Under which well-known container image repository will we publish the images?
```

Comment:
I do not think we need to decide this to merge this PR. We need to figure this put after talking to release engineering folks. Can we remove this open question from this PR?